althea_kernel_interface/src/lib.rs

```diff
@@ -30,7 +24,6 @@ mod exit_client_tunnel;
 mod exit_server_counter;
 mod exit_server_tunnel;
 mod get_neighbors;
-mod get_wg_pubkey;
```

Comment:
should we delete the entire `get_wg_pubkey` file?

Reply:
it doesn't seem to do anything anymore.